ccd-definition/CaseEvent/Submitted.json

```diff
@@ -219,5 +219,19 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "limitedUploadDocuments",
+    "Name": "Documents",
+    "Description": "Upload documents",
+    "DisplayOrder": 1,
```

Comment:
If this is one, should be placed next to 1 with display order. But probably should be 15 to mimic the doc placement?

Reply:
done